Automattic/wp-calypso
# Nextdoor: Update the connections screen styling

In #83227, we had added a placeholder image and text for Nextdoor on the connections screen. This PR updates that with the actual image and content.

Related to #83227

## Proposed Changes

* Update the connections screen styling for Nextdoor

## Testing Instructions

* Add the blog sticker to your site:
`add_blog_sticker( 'jetpack-social-nextdoor-connection', null, null, <YOUR_SITE_ID> );` 
* Goto `/marketing/connections/:site` in Calypso Blue
* Expand the Nextdoor connection along with the description _"Share posts with your local community with Nextdoor."_
* Confirm that you see the placeholder image and the text _"Connect with friends, neighbors, and local businesses by automatically sharing your posts to Nextdoor."_
* Confirm that the image is replaced with the one given in design - iv1KigPfttGxau8CQ1S0ZH-fi-3787_1078
* Goto `/jetpack-social/:site` on Calypso Green and confirm the above steps

<img width="588" alt="Screenshot 2023-11-02 at 4 03 10 PM" src="https://github.com/Automattic/wp-calypso/assets/18226415/a831cb3c-6c52-4e14-a253-77b916c05160">
<img width="1058" alt="Screenshot 2023-11-02 at 4 18 30 PM" src="https://github.com/Automattic/wp-calypso/assets/18226415/53c2c255-acc2-49ce-ac64-05559fa9224f">


## Pre-merge Checklist
